src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -383,7 +393,8 @@ def fuzz(self,
            corpus_directories,
            fuzz_timeout,
            artifact_prefix=None,
-           additional_args=None):
+           additional_args=None,
+           extra_env=None):
```

Comment:
looks like extra_env is not passed to LibFuzzerCommon.fuzz

Reply:
also need a test that shows LD_PRELOAD set to that path.

Reply:
>looks like extra_env is not passed to LibFuzzerCommon.fuzz

I think this was fixed in a more recent upload.

>also need a test that shows LD_PRELOAD set to that path.

I'll see how easy this is but I already test for this implicitly in the integration test I added.